scripts/items_required_logic.js

```diff
@@ -43,6 +49,9 @@ function itemsForRequirement(reqName) {
     var reqMet = true;
     var remainingProgress = 0;
   }
+  else {
+    console.log("Unrecognized reqName: " + reqName);
+  }
```

Comment:
I don't want to have any `console.log` statements in the codebase.

Reply:
This is an error case that can only be triggered by incorrect code. If you'd still like me to remove it, I will, but I think printing a line is better than failing silently.

Reply:
Can you throw an error instead?

Reply:
Sure, that works.